config/livewire.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Livewire Root Namespace
+    |--------------------------------------------------------------------------
+    |
+    | This value sets the root namespace of your application for the purpose
+    | of creating new Livewire Components via `php artisan make:livewire`
+    | commands. It will detect the application's namespace by default.
+    |
+    */
+
+    'namespace' => env('APP_NAMESPACE', 'App'),
+
+    /*
+    |--------------------------------------------------------------------------
+    | Livewire View Path
+    |--------------------------------------------------------------------------
+    |
+    | This value sets the path to new Livewire component views when creating
+    | them via `php artisan make:livewire` commands.
+    |
+    */
+
+    'view-path' => env('LIVEWIRE_VIEW_PATH', 'livewire'),
+
+    /*
+    |--------------------------------------------------------------------------
+    | Livewire Base URL
+    |--------------------------------------------------------------------------
+    |
+    | This value sets the path to Livewire JavaScript assets, for cases where
+    | your app's domain root is not the correct path. By default, Livewire
+    | will load its JavaScript assets from the app's "relative root".
+    |
+    */
+
+    'base_url'  => env('LIVEWIRE_BASE_URL', '/'),
+];
+/*  */
```

Comment:
Extra empty comment here?

Reply:
Not sure where that came from, haha, I have cleanup to do. 
I'm finally starting to grok some of the source here, I like where you're going with this library.

Reply:
Awesome to hear @defenestrator!